tracing-log/src/lib.rs

```diff
@@ -246,3 +256,136 @@ impl AsTrace for log::Level {
         }
     }
 }
+
+/// Extends log `Event`s to provide complete `Metadata`.
+///
+/// In `tracing-log`, an `Event` produced by a log (through `AsTrace`) has an hard coded
+/// "log" target and no file, line, or module_path attributes. This happens because `Event`
+/// requires its `Metadata` to be `'static`, while log records provide them with a generic
```

Comment:
Nit:
```suggestion
/// requires its `Metadata` to be `'static`, while [`log::Record`]s provide them with a generic
```
(and, add a link ref for `log::Record` at the bottom of the doc comment)